ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java

```diff
@@ -119,11 +119,21 @@ public void removeEntity(Entity entity) {
 	 */
 	private void moveMovables() {
 		for (IMovable movable : movables) {
-			movable.move(movable.getMoveBehaviour().getMoveVector());
+			movable.move(addDifficulty(movable.getMoveBehaviour().getMoveVector()));
```

Comment:
Right now EVERYTHING goes faster with the difficulty factor. So once my feature_multiSteering branch is finished and merged, the sensitivity of the accelerometer will be higher as well. So I don't think we want this to be the effect of the difficulty.
My idea was that the Platform has an AcceleratingMoveBehaviour instead of a ConstandSpeedMoveBehaviour, so that the moveVector gets increased slighlty by a certain value (can be passed in the constructor of that class) every time it is being get.